pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Linked Issues:
+| Jira Ticket 🎟️   | [Oceanwater-XXX](url) |
+| ----------- | ----------- |
+| EPIC ⚡| [Oceanwater-XXX](url) |
+| Github :octocat:  | # |
+
+
+## Summary of Changes
+* 
+
+## Test
+* 
```

Comment:
Dumb question: where is this file?

Reply:
i believe it lives top level of ow_simulator! @kmdalal

Reply:
I don't see this file, or the branch.  I'm approving based on the diff - looks good.

Reply:
@kmdalal You can see it if you go to the repo and switch the view from master to this hotfix branch. See screen shot (at the bottom). It's a completely new file so you wouldn't see it on master or anywhere else :) Will be merging!

![image](https://user-images.githubusercontent.com/3445834/102151728-4f996900-3e28-11eb-8ca7-be92535513fa.png)